src/main/java/com/krux/stdlib/utils/ExternalProperties.java

```diff
@@ -0,0 +1,48 @@
+package com.krux.stdlib.utils;
+
+import com.krux.stdlib.KruxStdLib;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Loads properties from an external file, set via the --property-file
+ * command-line option.
+ *
+ * For tests in your project,implement a helper class that reads the
+ * resource from test/resources.
+ */
+
+public class ExternalProperties {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger( ExternalProperties.class.getName() );
+
+    private Properties _kruxExternalProps;
+
+    public ExternalProperties() {
+        _kruxExternalProps = new Properties();
+        InputStream input;
+        if ( KruxStdLib.PROPERTY_FILE != null ) {
+            LOGGER.debug("loading properties from " + KruxStdLib.PROPERTY_FILE);
+            try {
+                input = new FileInputStream( KruxStdLib.PROPERTY_FILE );
+                _kruxExternalProps.load( input );
```

Comment:
`getResourceAsStream` will try to load the property file from the project's resources directory, which is not what is wanted here. The idea is to permit a path outside of the jar/project to be read, so it can can be managed by one's configuration management system.